### DISCUSSION TITLE
ch_PP-OCRv4_server_det文本检测模型的检测方向问题

### DISCUSSION THREAD

**Discussion by wzz981** [PaddlePaddle/PaddleOCR#12199] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：Windows10
- 版本号/Version：
          - Paddle：2.4.2  
         -  PaddleOCR：2.7 
         - 问题相关组件/Related components：文本检测

各位好，我在使用ch_PP-OCRv4_server_det这个文本检测模型针对户口本图片进行识别时，文本检测会把横向文字检测为纵向文字，请问可以设置文本检测模型只进行横向检测吗或者有其他更好的方法，有人遇到该问题，请帮忙解答，谢谢。

![image](https://github.com/PaddlePaddle/PaddleOCR/assets/95325500/f8266971-c44a-4bad-98a8-7c8a1735f4c4)

我们提供了AceIssueSolver来帮助你解答问题，你是否想要它来解答(请填写yes/no)?/We provide AceIssueSolver to solve issues, do you want it? (Please write yes/no):

请尽量不要包含图片在问题中/Please try to not include the image in the issue.

**Comment by sunfan1997** [comment]
我在身份证OCR识别中也遇到这个问题，自己重新标注横向文本后，会好一些。
